Packs/CSCDomainManager/README.md

```diff
@@ -0,0 +1,12 @@
+CSCDomainManager is a web-based portfolio management platform consolidating domains alongside social media usernames, SSL digital certificates, and DNS.
+
+As the world's leading corporate domain name registrar, CSC is the only company in the unique position to offer the most innovative, next generation domain management and security solutions coupled with online brand and fraud protection. DomainSec is a new, comprehensive platform for domain security. Powered by a correlation engine and machine learning technology, it brings together the variety of data sets and blocking networks only available to CSC, enabling threat intelligence based on domain security insights.
+
+We manage the most trafficked websites online. That's why at CSC, we provide you with highly-secure, intelligent, managed services to support your valuable domain portfolios. We work on your behalf to maximize your investment, and save you money. We have the largest global footprint of enterprise domains on the internet with 24x7x365 capabilities.We manage the most trafficked websites online. That's why at CSC, we provide you with highly-secure, intelligent, managed services to support your valuable domain portfolios. We work on your behalf to maximize your investment, and save you money. We have the largest global footprint of enterprise domains on the internet with 24x7x365 capabilities.
```

Comment:
This information is copied directly from the CSC web site.  
You need to rework this.  It has too much marketing.  It needs to be toned down. Maybe something like this:

CSC offers innovative, next generation domain management and security solutions coupled with online brand and fraud protection. DomainSec is a comprehensive platform for domain security. Powered by a correlation engine and machine learning technology, it brings together the variety of data sets and blocking networks enabling threat intelligence based on domain security insights.